manifests/install.pp

```diff
@@ -73,6 +81,15 @@
     subscribe   => Vcsrepo["${misp::install_dir}/app/files/scripts/mixbox"],
   }
 
+  exec {'pydeep build':
+    command     => '/usr/bin/python setup.py build && /usr/bin/python setup.py install',
+    cwd         => "${misp::install_dir}/app/files/scripts/pydeep/",
+    unless      => '/usr/bin/pip list | grep pydeep',
```

Comment:
Won't this prevent upgrades? Currently it's refreshonly and subscribing to the vcsrepo so I'd expect it's already idemptotent.